### DISCUSSION TITLE
Autoinstallable Distributions

### DISCUSSION THREAD

**Discussion by j4ccob** [uyuni-project/uyuni#8953] [Devel] [answered]
Hello,
I'm going to create a distribution, but I need to mount as a loop a iso, like We did on the Legacy enviroment

I would like to ask is it some process how to make a mount on the podman ? 

Or I have to mount it on the server and just a copy files in podman to path, what I need? 

```
uyuni-server:/ # mount -o ro,loop,auto /var/spacewalk/iso/openSUSE-Leap-15.5-DVD-x86_64-Build491.1-Media.iso /srv/iso/openSUSE-15-SP5-Server
mount: /srv/iso/openSUSE-15-SP5-Server: mount failed: Operation not permitted.

```

**Comment by admd** [comment, marked as answer]
`mgradm` will come with a command `distor` to help you with this. You wouldn't need to do it manually.
